online_store/gt_sport_v2.js

```diff
@@ -91,15 +91,22 @@ function productDescriptionDisplay() {
 };
 productDescriptionDisplay();
 
-const btnEl = productEl.querySelector("button");
-btnEl.addEventListener('click', addProductsToCart);
-productListEl.appendChild(productEl);
+function addProductsToCart() {
+  for (let i = 0; i < sportProducts.length; i++) {
+    shoppingCart.push(sportProducts[i]);
+  };
+  console.log(shoppingCart);
+};
+
+const btnEl = document.createElement("button");
```

Comment:
this button should be created not in here, but right after the product description, inside you're for loop.
addProductsToCart - should be a function with one parameter. and it should looks somethign like that:
```javascript
function addProductsToCart(product) {
    shoppingCart.push(product);
};
```
Otherwise this function (as it is right now) would add all element to shopping cart each time you click on any button, because you have loop inside.

Reply:
Changed, done. Thx.